api/src/certification/session-management/domain/usecases/create-companion-alert.js

```diff
@@ -0,0 +1,21 @@
+import { withTransaction } from '../../../../shared/domain/DomainTransaction.js';
+import { CertificationCompanionLiveAlert } from '../models/CertificationCompanionLiveAlert.js';
+
+/**
+ * @typedef {import('../usecases/index.js').CertificationCompanionAlertRepository} CertificationCompanionAlertRepository
+ */
+
+export const createCompanionAlert = withTransaction(
+  /**
+   * @param {Object} params
+   * @param {number} params.assessmentId
+   * @param {CertificationCompanionAlertRepository} params.certificationCompanionAlertRepository
+   **/
+  async function ({ assessmentId, certificationCompanionAlertRepository }) {
+    const companionAlert = new CertificationCompanionLiveAlert({
+      assessmentId,
+      status: 'ONGOING',
```

Comment:
CertificationCompanionLiveAlertStatus.ONGOING (autant utiliser l'enumeration cree pour l'occaz ce qui evite les [magic strings](https://dev.to/ruben_alapont/magic-numbers-and-magic-strings-its-time-to-talk-about-it-ci2)).

Autre version que j'aime bien, mais ptet overkill, une [factory](https://beratyesbek.medium.com/item-1-effective-java-consider-static-factory-methods-instead-of-constructors-7ff773804aaa) (j'ai pas cherche d'article en node) : `CertificationCompanionLiveAlert.createOngoing()`